pyaedt/application/Variables.py

```diff
@@ -961,22 +1084,25 @@ def format(self, format):
 
     @aedt_exception_handler
     def __mul__(self, other):
-        """Multiply the Variable with a number or another Variable and return a new object
+        """Multiply the variable with a number or another variable and return a new object.
 
         Parameters
         ---------
-        other : numbers.Number or Variable
-            object to be multiplied
+        other : numbers.Number or variable
+            Object to be multiplied.
 
         Returns
         -------
-        Variable
+        type
+            Variable.
 
         Examples
         --------
         >>> from pyaedt.application.Variables import Variable
 
-        Multiply 'Length' by unitless ('None') to obtain 'Length'. A numerical value is also considered to be unitless
+        Multiply '`Length1`' by unitless (`'None'`) to obtain `'Length'`. 
```

Comment:
```suggestion
        Multiply ``'Length1'`` by unitless ``'None'``` to obtain ``'Length'``. 
```